Tests/helper.py

```diff
@@ -95,7 +95,7 @@ def assert_image_equal(a, b, msg=None):
             except Exception:
                 pass
 
-        assert False, msg or "got different content"
+        assert a.tobytes() == b.tobytes(), msg or "got different content"
```

Comment:
Why make this change? `a.tobytes()` and `b.tobytes()` have already been compared a few lines earlier, so we know this is false.

Reply:
It gives a better error message.
```python
import pytest

def test_assert_false():
    assert False, "message"

def test_assert_compare():
    assert b"abc" == b"abb", "message"
```
```python
================ FAILURES =================
____________ test_assert_false ____________

    def test_assert_false():
>       assert False, "message"
E       AssertionError: message
E       assert False

Tests/test_assert.py:4: AssertionError
___________ test_assert_compare ___________

    def test_assert_compare():
>       assert b"abc" == b"abb", "message"
E       AssertionError: message
E       assert b'abc' == b'abb'
E         At index 2 diff: b'c' != b'b'
E         Use -v to get more diff

Tests/test_assert.py:7: AssertionError
```

Reply:
How about https://github.com/Yay295/Pillow/pull/1 instead?

Reply:
Nice idea.

Reply:
In https://github.com/Yay295/Pillow/actions/runs/3810393071/jobs/6482448151#step:8:1241, you can see what happens when this fails now.

...the output is very large.

Reply:
Apparently this is due to the "-v" option. ".ci/test.sh" uses this when running tests, but the makefile doesn't (except for "valgrind" which uses "-vv"), so I hadn't encountered this locally.

Reply:
It looks like there's an open issue on pytest's GitHub for this: https://github.com/pytest-dev/pytest/issues/5285

Reply:
I found a way to temporarily set the verbosity to 0. d75b31b6eae1bc2bfb77bb425e5d043a02a952ae